src/DynamoCoreWpf/Properties/Resources.en-US.resx

```diff
@@ -134,19 +134,19 @@
     <comment>To indicate Dynamo is up to date</comment>
   </data>
   <data name="ChangeScaleFactorPromptCaptionText" xml:space="preserve">
-    <value>What is the range of geometry sizes that you will be working on?</value>
+    <value>To ensure accurate geometry rendering and calculation, select the range of geometry sizes that you will be working on:</value>
   </data>
   <data name="ChangeScaleFactorPromptDescriptionBox" xml:space="preserve">
     <value>Numerical range shown here denote numbers in the following unit:</value>
   </data>
   <data name="ChangeScaleFactorPromptDescriptionContent0" xml:space="preserve">
-    <value>Use this working range to model objects the size of a city block that can stretch a few kilometers and with no more accuracy than a millimeter.</value>
+    <value>Use this working range to model in values that fall between 0.000001 to 100 units</value>
```

Comment:
You could change this to `Use this working range to model in values that fall between {0} to {1} units` and substitute values for `{0}` and `{1}` from `ScaleRange.Item1` and `ScaleRange.Item2` properties in `ChangeScaleFactorPrompt` as mentioned below. Then do the similar change for all other descriptions.